streams/src/test/java/org/apache/kafka/streams/integration/KTableKTableForeignKeyInnerJoinMultiIntegrationTest.java

```diff
@@ -83,9 +83,9 @@
     public static void beforeTest() throws Exception {
         //Use multiple partitions to ensure distribution of keys.
         CLUSTER.createTopic(TABLE_1, 3, 1);
-        CLUSTER.createTopic(TABLE_2, 7, 1);
+        CLUSTER.createTopic(TABLE_2, 5, 1);
         CLUSTER.createTopic(TABLE_3, 7, 1);
-        CLUSTER.createTopic(OUTPUT, 7, 1);
+        CLUSTER.createTopic(OUTPUT, 11, 1);
```

Comment:
After fixing copartitioning, I made all the topics co-prime to prove that the copartitioning works now.